src/node_binding.h

```diff
@@ -21,6 +21,10 @@ enum {
   NM_F_DELETEME = 1 << 3,
 };
 
+// Make sure our internal values match the public API's values.
+static_assert(static_cast<int>(NM_F_LINKED) ==
+              static_cast<int>(node::ModuleFlags::kLinked));
```

Comment:
This needs an assert message (i.e., `static_assert(cond, "cond failed");`)

Reply:
Thanks, done. Interesting that it didn’t fail locally. :)

Reply:
You probably compiled it as C++17, the message is optional in that version of the language.